src/routes/Onboarding/BrowserOnboardingRoutes.tsx

```diff
@@ -468,10 +470,8 @@ function OnboardingConclusion() {
         <p className="my-4">
           We highly encourage you to{' '}
           <a
-            onClick={openExternalBrowserIfDesktop(
-              `${VITE_KC_SITE_BASE_URL}/modeling-app/download/nightly`
-            )}
-            href="https://zoo.dev/modeling-app/download/nightly"
```

Comment:
Did a bit of extra cleanup, and thi shouldn't be nightly IMO

Reply:
Whaaa good catch I have no idea why I had that set to nightly!